src/ui/src/index.html

```diff
@@ -46,25 +46,39 @@ <h1 id="app-name"></h1>
     </app-root>
   </div>
   <script>
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+  (function(i, s, o, g, r, a, m) {
```

Comment:
Why did you uncompress the Google Analytics method and change the indentation of the script?